menu.js

```diff
@@ -300,3 +309,38 @@ module.exports.setApplicationMenu = (win) => {
 	const menu = Menu.buildFromTemplate(menuTemplate);
 	Menu.setApplicationMenu(menu);
 };
+
+const iconPath = path.join(__dirname, "assets", "youtube-music-tray.png");
+const example = "Example: 'socks5://127.0.0.1:9999'";
+function setProxy(item, win) {
+	let options = {
+		title: 'Set Proxy',
+		label: 'Enter Proxy Address: (leave empty to disable)',
+		value: config.get("options.proxy") || example,
+		type: 'input',
+		inputAttrs: {
+			type: 'url'
+		},
+		icon: iconPath,
+		customStylesheet: "dark",
+		width: 450,
+	};
+	//TODO: custom bar on prompt need testing on macOS
+	if (!is.macOS()) {
+		Object.assign(options, {
+			frame: false,
+			customScript: path.join(__dirname, "plugins", "in-app-menu", "prompt-custom-titlebar.js"),
```

Comment:
We should probably only style the prompt with the custom title bar if the `in-app-menu` plugin is enabled

Reply:
I just think it looks massively better with the custom titlebar. (atleast on windows)
And I dont really see the point in not enabling it tbh